.github/workflows/build-and-save-contract.yml

```diff
@@ -0,0 +1,56 @@
+name: Build and save DDC SC binaries
+on: [push]
+
+jobs:
+  build-and-save:
```

Comment:
What about just `build`?

Reply:
The jobs builds and saves 
I thought this is the point of workflow

Reply:
The artifact (result of any build) should be stored somewhere. Otherwise, why to build?
Sometimes, artifact is uploading to 3rd party (docker image to registry, web app bundle to S3, etc.). In this case it makes sense to mention this. But when the CI server's storage is using, usually `save` is skipping.

Reply:
Ok, got it will remove

Reply:
fixed